Sources/SourceKitLSP/SourceKitServer.swift

```diff
@@ -2000,27 +2000,28 @@ extension SourceKitServer {
     else {
       return []
     }
-    let occurs = index.occurrences(ofUSR: data.usr, roles: .calledBy)
-    let calls = occurs.compactMap { occurrence -> CallHierarchyIncomingCall? in
-      guard let location = indexToLSPLocation(occurrence.location),
-        let related = occurrence.relations.first
-      else {
-        return nil
+    let callableUsrs = [data.usr] + index.occurrences(relatedToUSR: data.usr, roles: .accessorOf).map(\.symbol.usr)
+    let callOccurrences = callableUsrs.flatMap { index.occurrences(ofUSR: $0, roles: .calledBy) }
```

Comment:
Does this ignore eg. `let foo = someFunc`? I know it's not strictly correct, but IMO it's still useful to include non-calls in call hierarchy. Open to opinions though.